agent/tool-scripts/base-tool

```diff
@@ -186,6 +186,9 @@ virsh-migrate)
 	longoptions="${longoptions},vm:"
 	tool_package_name="libvirt-client"
 	;;
+vmstat)
+	tool_package_name="procps-ng"
```

Comment:
procps-ng needs to be deleted from the spec file.

Reply:
Fixed.